What is a Temporal Cron Job -> updated diagrams, sm additons per feedback (#620)
temporalio/documentation: docs/content/what-is-a-temporal-cron-job.md

A Temporal Cron Job is the series of Workflow Executions that occur when a Cron Schedule is provided in the call to spawn a Workflow Execution.

<CenteredImage
imagePath="/diagrams/temporal-cron-job.svg"
imageSize="100"
title="Temporal Cron Job timeline"
/>

A Temporal Cron Job is very similar to a cron job.

> Just as a Linux based cron job accepts a command and a schedule on which to execute that command, a Cron Schedule can be provided with the call to spawn a Workflow Execution.

Reviewer: ```suggestion
Just as a Linux-based cron job accepts a command and a schedule on which to execute that command, a Cron Schedule can be provided with the call to spawn a Workflow Execution.
```

* * * * *
```

For example, "15 8 \* \* \*" causes a Workflow Execution to spawn daily at 8:15 AM UTC.
Use the [crontab guru site](https://crontab.guru/) to test your cron expressions.

**`robfig` predefined schedules**

You can also pass any of the [predefined schedules](https://pkg.go.dev/github.com/robfig/cron#hdr-Predefined_schedules) or [intervals](https://pkg.go.dev/github.com/robfig/cron#hdr-Intervals) described in the [`robfig/cron` documentation](https://pkg.go.dev/github.com/robfig/cron).

> For example, "@weekly" would cause the a Workflow Execution to spawn once a week, midnight between Sat/Sun.

Reviewer: ```suggestion
For example, "@weekly" causes a Workflow Execution to spawn once a week at midnight between Saturday and Sunday.
```

### Stop Cron Job

> A Temporal Cron Job will not stop spawning Runs until it has been Terminated, or the [Workflow Execution Timeout](/docs/content/what-is-a-workflow-execution-timeout) is reached.

Reviewer: ```suggestion
A Temporal Cron Job does not stop spawning Runs until it has been Terminated or until the [Workflow Execution Timeout](/docs/content/what-is-a-workflow-execution-timeout) is reached.
```

> A Cancellation Request will only affect the current Run.

Reviewer: ```suggestion
A Cancellation Request affects only the current Run.
```
